### ISSUE TITLE
Remoting with Azure SpatialAnchors session constantly throws RemotingHandlerCore_GetAnchorExportSufficiency_Failed

### ISSUE THREAD

**Issue by xiaoguid** [microsoft/MixedReality-HolographicRemoting-Samples#49]
Hello,

I have recently added Azure SpatialAnchors support on top of remoting (2.7.1, C++/winrt) to our product.

After starting an Azure session (CloudSpatialAnchorSession::Start), I'm constantly getting:
"Failed to get availability status: RemotingHandlerCore_GetAnchorExportSufficiency_Failed : Remote call request failed".

This actually comes from the Azure callback logs as well as debug output but I already know those errors originate from the remoting library as remoting practically replaces all local interfaces. So I'm posting the issue here.

The error happens every 2-3 seconds and is quickly followed by a CloudSpatialAnchorSession::SessionUpdated callback with all metrics dropping to 0 - which sometimes pick up a few seconds later only to be 0 again after the next error.

Needless to say that having all metrics drop to 0 every two seconds practically renders Azure cloud anchors with remoting useless for us.. you can't properly save anchors until RecommendedForCreateProgress() >= 1.0f.
There is nothing special about the Azure session (4 lines of code), I create the session and call Start() only after successfully connecting to the remote headset.

Since in-between errors everything sorta works (I do get valid metrics for a few seconds before getting the error again and so on) - I was wondering if this is the expected (broken) behavior or I might be missing something?

Otherwise there seems to be a bug where Azure calls GetAnchorExportSufficiency internally and remoting 70% fails the call.

<details>
<summary>Example Debug Logs - only after starting an Azure session</summary>
Exception thrown at 0x00007FFBECA14F99 in .exe: Microsoft C++ exception: [rethrow] at memory location 0x0000000000000000.
Exception thrown at 0x00007FFBECA14F99 in .exe: Microsoft C++ exception: winrt::hresult_error at memory location 0x00000018E36FF550.
Exception thrown at 0x00007FFBECA14F99 in .exe: Microsoft C++ exception: winrt::hresult_error at memory location 0x00000018E09FEFC0.

-----------------------------------------

HololensLink.cpp(1602): CloudSessionLog: Failed to get availability status: RemotingHandlerCore_GetAnchorExportSufficiency_Failed : Remote call request failed

-----------------------------------------

Exception thrown at 0x00007FFBECA14F99 in .exe: Microsoft C++ exception:
Microsoft::Holographic::AppRemoting::RemoteCallException at memory location 0x00000018E3AFE510.
Exception thrown at 0x00007FFBECA14F99 in .exe: Microsoft C++ exception: [rethrow] at memory location 0x0000000000000000.

-----------------------------------------

Exception thrown at 0x00007FFBECA14F99 in .exe: Microsoft C++ exception: 
Microsoft::Holographic::AppRemoting::RemoteCallException at memory location 0x00000018E3AFE510.
Exception thrown at 0x00007FFBECA14F99 (KernelBase.dll) in .exe: WinRT originate error - 0x80004005 : 'RemotingHandlerCore_GetAnchorExportSufficiency_Failed : Remote call request failed'.

-----------------------------------------

Exception thrown at 0x00007FFBECA14F99 in .exe: Microsoft C++ exception: winrt::hresult_error at memory location 0x00000018E3AF9EF0.
Exception thrown at 0x00007FFBECA14F99 in .exe: Microsoft C++ exception: winrt::hresult_error at memory location 0x00000018E3BFF410.
</details>

Much appreciated,
Daniel

**Comment by xiaoguid** [comment]
Hi @FlorianBagarMicrosoft , is there any feedback on this?
Repro'ing is very easy, just start a session while remoting.
Apologies if this does not seem to be a remoting issue.
Should I post this in the Azure SpatialAnchors repo instead?

**Comment by NPohl-MSFT** [comment]
Hi @xiaoguid,

thanks for raising this! It indeed rather looks like a remoting issue.
Just to be sure - is this something that worked for you in an earlier version (e.g. remoting 2.7.0)? 
Or did you just add the ASA support on top of the latest remoting version?

Thanks,
Norman

**Comment by xiaoguid** [comment]
@NPohl-MSFT
Just added ASA support on top of our existing remoting app (updated to 2.7.1).
Using the latest Microsoft.Azure.SpatialAnchors.WinRT (2.10.2).

**Comment by NPohl-MSFT** [comment]
Hi @xiaoguid,
thanks again for raising this.

We were able to reproduce the issue and will fix it with the next update.

**Comment by xiaoguid** [comment]
@NPohl-MSFT that's great to hear, thanks for looking into this.
Do you have a rough estimate for the next update release?
If the fix is expected soon I'd rather postpone shipping my workaround.
Much appreciated!

**Comment by maapp** [comment]
Hey @xiaoguid,

we released 2.7.2 yesterday which includes the fix for the issue.

Bye,
Marc
